Domains/Smartmetering/smartmeteringwebservices/SetDeviceLifecycleStatus.md

```diff
@@ -0,0 +1,14 @@
+## SetDeviceLifecycleStatus request
```

Comment:
This probably shouldn't be added here, since this is not an operation from `SmartMeteringManagement.wsdl`, but from the core `DeviceManagement.wsdl`. (Same for the response.)